lib/reek/configuration/configuration_file_finder.rb

```diff
@@ -110,6 +110,7 @@ def find_in_dir(dir)
         # @return [undefined]
         #
         def escalate_too_many_configuration_files(found, directory)
+          # Follow up TODO: Shouldn't this rather raise a ConfigFileException?
```

Comment:
Yes, it should. What is that `exit 1` even doing there?

Reply:
Just noticed that the whole method goes in [1315](https://github.com/troessner/reek/pull/1315/files#diff-8b5f007c73578c73281c4d09e44bab4cL112) so that problem solves itself :D